app/src/main/java/uk/gov/android/securestore/AccessControlLevel.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.android.securestore
+
+enum class AccessControlLevel {
```

Comment:
Probably not one for this PR as we spoke about how we're not really handling this at the moment: Are we able to get some KDoc on this to help explain what the different states mean?

Reply:
Yes, will certainly add some into the next PR